Cargo.toml

```diff
@@ -37,7 +37,7 @@ iocuddle = "0.1"
 
 [dev-dependencies]
 dirs = "3.0"
-kvm-bindings = "0.4"
+kvm-bindings = "0.5"
```

Comment:
I think this should be merged into whichever commit depends on this change, as well as a note describing why you bumped the version.

Reply:
I also bumped `kvm-ioctls` in this commit too, as to avoid the naming conflict between `kvm-bindings` and `kvm-ioctls`